darksky/request_manager.py

```diff
@@ -25,14 +25,22 @@ def make_request(self, url: str, **params):
 
 
 class RequestMangerAsync(BaseRequestManger):
+    def __init__(self, gzip: bool, client_session: aiohttp.ClientSession = None):
```

Comment:
It will raise an error "NameError: name 'aiohttp' is not defined"

Reply:
You are right, the type hint has to be removed if the import is happening in __init__